run

```diff
@@ -24,5 +24,6 @@ export STD_LIBRARY_PATH="${SCRIPT_PATH}/lib/stdlib.c"
 test -t 2 && export COLOR="on"
 
 # execute the compiler with the provided arguments and return the exit code
-java -jar "$EXE_PATH" "$@"
+# -ea enables assertions
+java -jar -ea "$EXE_PATH" "$@"
```

Comment:
Should we really enable assertions here (this script is used for benchmarks)?

Reply:
Assertions should be enabled at least for the test script, I think, but disabled for the final version of the compiler. Regarding the run script here, we could either separate it from the test script and/or remove the assertions later, once we are confident enough in the correctness.

I wonder whether enabled assertions might simplify pinning down bugs found by our supervisors?

Reply:
I agree that we should definitely enable assertions in the `test` script (which doesn't currently exist in master). For now we can also leave them enabled in `run`. I'm not sure that should be permanent though. Even ignoring the performance penalty, it risks test failures because of incorrect assertions.